Assets/StreamingAssets/Data/Furniture.xml

```diff
@@ -46,7 +46,7 @@
         <PathfindingWeight>1</PathfindingWeight>
         <Width>1</Width>
         <Height>1</Height>
-        <LinksToNeighbours>false</LinksToNeighbours>
+        <LinksToNeighbours></LinksToNeighbours>
```

Comment:
We can just as well add it back if we want to link doors. This way it looks like it's not optional or that maybe empty is a special value.